spec/system/manage_users/add_user_spec.rb

```diff
@@ -4,6 +4,9 @@
   include_context 'when logged in user is admin'
 
   before do
+    allow(NotifyMailer).to receive(:access_granted_email) {
```

Comment:
Wonder if it might be worth having a shared helper for this particular stubb? Might DRY it out a bit we could pass the template key in.

Just a thought not blocking for this PR :)